client/www/pages/intern/platform_demo.tsx

```diff
@@ -0,0 +1,462 @@
+  -H "Authorization: Bearer $PLATFORM_TOKEN" \\
+  -H "Content-Type: application/json" \\
+  -d '{"title": "my cool app"}'  
+`.trim();
+};
+
+const exampleInit = (appId: string): string => {
+  return `
+import { init } from '@instantdb/react'; 
+init({appId: "${appId}"}); // 🎉
+  `.trim();
+};
+
+const exampleSchemaGen = (appId: string) => {
+  return `
+import { i } from '@instantdb/core';
+const g = i.graph(
+  appId,
+  {
+    posts: i.entity({
+      title: i.string(),
+      body: i.string(),
+    }),
+    comments: i.entity({
+      body: i.string(),
+    }),
+  },
+  {
+    commentPosts: {
+      forward: {
+        on: 'comments',
+        has: 'one',
+        label: 'post',
+      },
+      reverse: {
+        on: 'posts',
+        has: 'many',
+        label: 'comments',
+      },
+    },
+  }
+);
+JSON.stringify(g, null, 2) // this is the schema you can push!
+`.trim();
+};
+const exGraph = (appId: string) => {
+  const g = i.graph(
+    appId,
+    {
+      posts: i.entity({
+        title: i.string(),
+        body: i.string(),
+      }),
+      comments: i.entity({
+        body: i.string(),
+      }),
+    },
+    {
+      commentPosts: {
+        forward: {
+          on: 'comments',
+          has: 'one',
+          label: 'post',
+        },
+        reverse: {
+          on: 'posts',
+          has: 'many',
+          label: 'comments',
+        },
+      },
+    }
+  );
+  return g;
+};
+
+const exampleSchemaPushCurl = (token: string, appId: string): string => {
+  return `
+export PLATFORM_TOKEN="${token}"
+export APP_ID="${appId}"
+curl -v -X POST "${config.apiURI}/superadmin/apps/$APP_ID/schema/push/apply" \\
+  -H "Authorization: Bearer $PLATFORM_TOKEN" \\
+  -H "Content-Type: application/json" \\
+  -d '${JSON.stringify({ schema: exGraph(appId) }, null, 2)}'  
+`.trim();
+};
+
+const examplePermsCurl = (token: string, appId: string): string => {
+  return `
+export PLATFORM_TOKEN="${token}"
+export APP_ID="${appId}"
+curl -v -X POST "${config.apiURI}/superadmin/apps/$APP_ID/perms" \\
+  -H "Authorization: Bearer $PLATFORM_TOKEN" \\
+  -H "Content-Type: application/json" \\
+  -d '${JSON.stringify(
+    { code: { posts: { allow: { create: 'false' } } } },
+    null,
+    2
+  )}'  
+`.trim();
+};
+
+function AppStage({
+  token,
+  app,
+  setApp,
+}: {
+  token: string;
+  app: { id: string };
+  setApp: any;
+}) {
+  const [transferEmail, setTransferEmail] = useLocalStorage<string>(
+    '__platform_demo_email'
+  );
+  const [transferResult, setTransferResult] = useState<any>();
+  const [cancelTransferResult, setCancelTransferResult] = useState<any>();
+  const [schemaPushResult, setSchemaPushResult] = useState<any>();
+  const [permsResult, setPermsResult] = useState<any>();
+  return (
+    <div>
+      <h2>2. Transfer Apps</h2>
+      <p>
+        To transfer an app, you need to know the user's email. Pop it in here:
+      </p>
+      <TextInput
+        value={transferEmail || 'stopa@instantdb.com'}
```

Comment:
The default `stopa@instantdb.com` shows up here, but it's undefined in the body below until you edit the input.

Idea for `useLocalStorage`--add another argument for a default value if localstorage is empty.